Source/Core/InputCommon/ControllerEmu/Setting/NumericSetting.h

```diff
@@ -158,10 +158,13 @@ class SettingValue
 public:
   ValueType GetValue() const
   {
-    if (IsSimpleValue())
-      return m_value;
-    else
-      return m_input.GetState<ValueType>();
+    // Only update dynamic values when the input gate is enabled.
+    // Otherwise settings will all change to 0 when focused is lost.
```

Comment:
"focused" -> "focus"

Reply:
I orignally had focus and that didn't sound right for some reason.. :P